src/storage/v2/disk/storage.cpp

```diff
@@ -402,43 +400,57 @@ VerticesIterable DiskStorage::DiskAccessor::Vertices(LabelId label, View view) {
   index_deltas_storage_.emplace_back(std::list<Delta>());
```

Comment:
You can modify this too, no need to send std::list<Delta>(), you can just do:
`index_deltas_storage_.emplace_back();`